### DISCUSSION TITLE
How easy is it to migrate from Rojo?

### DISCUSSION THREAD

**Discussion by RuizuKun-Dev** [argon-rbx/argon#60] [Q&A] [answered]
So how difficult is it to migrate from Rojo?

**Comment by DervexDev** [comment, marked as answer]
Even though Argon uses different field names in `project.json` it is backward-compatible with one exception:
* If you have `placeId` in your `project.json` you need to move it to `servePlaceIds` array

All Rojo file types are supported by Argon as well with one exception:
* If you have any `init.csv` files you need to rename them to `.src.csv`

And that should be it.

**Reply by RuizuKun-Dev** [reply]
thank you! that sounds easy enough i'll give it a try and give feedback
